src/plugins/mf_priority.cpp

```diff
@@ -108,6 +108,13 @@ int64_t priority_calculation (flux_plugin_t *p,
     fshare_factor = b->fairshare;
     queue_factor = b->queue_factor;
 
+    priority = round ((fshare_weight * fshare_factor) +
+                      (queue_weight * queue_factor) +
+                      (urgency - 16));
+
+    if (fshare_factor < 0.00000)
+            return FLUX_JOB_PRIORITY_MIN;
+
     priority = round ((fshare_weight * fshare_factor) +
```

Comment:
Maybe I'm reading the diff wrong, but is `priority` re-calculated here? Perhaps the additional one above wasn't meant to be added to the file.

Reply:
Yup, you're completely right - I must have made a mistake during my rebase and obviously things got a little haywire 🤦‍♂️ There should only be one calculation, and the test for `fshare_factor` is in fact before the priority calculation, as you mentioned above. I'll push up a fix, I'm sorry about that!

Reply:
Ah, that is what I figured :+1: